<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SonoForm</class>
 <widget class="QMainWindow" name="SonoForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sono Felice Country club</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #76528B;
color:#CBCE91;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,5">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="2,1,1,1,1,1">
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>예약 일자</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>선호 시간</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Old English Text MT</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sono Felice Country Club</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QDateEdit" name="idate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLineEdit" name="ipassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QLabel" name="label_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>까지</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="ito_time">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>전체</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19시</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="ifm_time">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>전체</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18시</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19시</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="iid">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>선호 코스</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>부터</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QComboBox" name="icourse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>비발디파크 East</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>비발디파크 West</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>델피노</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>비발디파크 Mountain</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="iproc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>21</pointsize>
            </font>
           </property>
           <property name="text">
            <string>예약하기</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Count&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Down&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="ilcdNumber">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="digitCount">
              <number>8</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="iresult"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>iid</tabstop>
  <tabstop>ipassword</tabstop>
  <tabstop>idate</tabstop>
  <tabstop>ifm_time</tabstop>
  <tabstop>ito_time</tabstop>
  <tabstop>icourse</tabstop>
  <tabstop>iresult</tabstop>
  <tabstop>iproc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
